src/MetaMorpho.sol

```diff
@@ -650,7 +658,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
     /// @dev Returns the vault's balance the market defined by `marketParams`.
     function _supplyBalance(MarketParams memory marketParams) internal view returns (uint256) {
-        return MORPHO.expectedSupplyBalance(marketParams, address(this));
+        return MORPHO.expectedSupplyAssets(marketParams, address(this));
```

Comment:
Nitpick: the function name should match the underlying function's name called. This also solves the name weirdness that right now could seem to imply that you are going to **supply** (as an action) instead of retrieving the value.